include/plugin.hpp

```diff
@@ -306,6 +324,9 @@ class ibmpowernv_plugin
     std::thread collection_thread;
     /// points in time when measurements have been taken
     std::vector<scorep::chrono::ticks> times_;
+    /// points in time when measurements have been taken
+    std::vector<std::chrono::time_point<std::chrono::system_clock>> system_times_;
```

Comment:
either `steady_clock` or `high_resolution_clock`, b/c `system_clock` can theoretically run backwards or do other wonky stuff

(the clean solution would be to use the OCC 512 MHz-based timestamp, which was exact in test measurements)

Reply:
feel free to change. I did not know that this timer existed.

Reply:
and to be honest, I don't know how to use it ^^.